index.html

```diff
@@ -0,0 +1,152 @@
+                const target = e.target.closest(".button"); // Or any other selector.
+
+                if(target){               
+                    shelf.removeBook(target.id);
+                }
+        });   
+        var date = new Date();
+        var month = date.toLocaleString('default', { month: 'long' });
+        const formatMap = {
+            mm: date.toLocaleString('default', { month: 'long' }) ,
+            dd: date.getDay(),
+            yy: date.getFullYear()
+           
+        };
+        var div = document.getElementById('date-time');
+
+        function get_ordinal_suffix(i) {
+            var j = i % 10,
+                k = i % 100;
+            if (j == 1 && k != 11) {
+                return i + "st";
+            }
+            if (j == 2 && k != 12) {
+                return i + "nd";
+            }
+            if (j == 3 && k != 13) {
+                return i + "rd";
+            }
+            return i + "th";
+        }
+        
+        function time(date) {
+            var date = new Date();
+
+            var d = new Date();
+            var s = date.getSeconds();
+            var m = date.getMinutes();
+            var h = date.getHours();
+            div.textContent =   date.toLocaleString('default', { month: 'long' }) +" " + get_ordinal_suffix(date.getDay())+" " + date.getFullYear()  +" "+("0" + h).substr(-2) + ":" + ("0" + m).substr(-2) + ":" + ("0" + s).substr(-2) +" "+ (h >= 12 ? 'pm' : 'am');
+        }
+
+        setInterval(time, 1000);
+
+        const links = document.querySelectorAll('.links'); /// create array of element objects
+
+        list.classList.toggle("show");
+        form.classList.toggle("show"); 
+        contact.classList.toggle("show");
+        list.style.display = "block";  
+        links.forEach(link => { // lopp through them
+            link.addEventListener('click', function handleClick(event) {
+             
+
+                var list = document.getElementById('list'); 
+                var form = document.getElementById('form');
+                var contact = document.getElementById('contact'); 
+
+                 if(this.id==="list-link"){                        
+                    list.style.display = "block";                       
+                    form.style.display = "none";   
+                    contact.style.display = "none";   
+                 }
+                
+                 if(this.id==="add-link"){          
+                    list.style.display = "none";                       
+                    form.style.display = "block";   
+                    contact.style.display = "none";   
+                 }
+
+                 if(this.id==="contact-link"){   
+                    list.style.display = "none";                       
+                    form.style.display = "none";   
+                    contact.style.display = "block";   
+                 } 
+              
+            }); 
+        });    
+
+    </script>
```

Comment:
- [ ] It would be nice if you put all these codes in a separate JavaScript file and then provide the link to that file as the value of the src attribute for the script tag. This would make your work look tidy, it is easy to debug, and you would be separating logic from markup. It is also one of the best industry practices when writing JavaScript codes. Please transfer the codes to a new JavaScript file. You can read this [article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) provided by Microverse to learn more about JavaScript best practices. 👍 😎